draft-huitema-dprive-dnsoquic.md

```diff
@@ -69,53 +86,49 @@ TCP and provides more efficient error corrections than UDP. DNS over QUIC
 (DNS/QUIC) has privacy properties similar to DNS over TLS specified in RFC7858,
 and performance similar to classic DNS over UDP.
 
-{mainmatter}
+--- middle
 
 # Introduction
 
-Domain Name System (DNS) concepts are specified in [@!RFC1034].  This
-document presents a mapping of the DNS protocol [@!RFC1035] over QUIC
-transport [@!I-D.ietf-quic-transport] [@!I-D.ietf-quic-tls].  The
+Domain Name System (DNS) concepts are specified in {{!RFC1034}}.  This
+document presents a mapping of the DNS protocol {{!RFC1035}} over QUIC
+transport {{!I-D.ietf-quic-transport}} {{!I-D.ietf-quic-tls}}.  The
 goals of this mapping are:
 
 1.  Provide the same DNS privacy protection as DNS over TLS (DNS/TLS)
```

Comment:
Suggest that we use an acronym of the form Do* instead of DNS/*  throughout because that is what is used in practice and other RFCs these days.....

Reply:
The abbreviation that I hear most is "DoQ". Paull Hoffman will object that DNS/HTTP3/QUIC is also DoQ, so I refrained using that so far. Would you also change the references to DNS/TLS and use DoT?

Reply:
Also, in the initial draft we recommended the ALPN "dq". Do you suggest changing that to "doq"? I did not change it because I assumed that prototype implementations were using "dq" and did not want to mess the interop. What is the ALPN for DoT?

Reply:
Agree that I have heard DoQ used and that is what I was thinking of.  If Paul objects maybe we could suggest DoQ2 and DoQ3 to differentiate?  And yes, I would change DNS/TLS to DoT and we could even reference Paul's draft that defines it: https://datatracker.ietf.org/doc/draft-ietf-dnsop-terminology-ter/. The ALPN for DoT is 'dot', but I take your point about interop....... I think we might be early enough to get away with changing it though?

Reply:
OK. I have replaced DNS/QUIC by DoQ and DNS/TLS per DoT, added the reference to the terminology draft, and set the ALPN to "doq". There is no possible ALPN conflict with DNS over HTTP3, since the ALPn used for that would be "h3". If Paul protests, we will have to convince him that DoH3 is a fine acronym for DNS over HTTP3.

Reply:
I also added a section on Address Validation, because that's somewhat important.